src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
```

Comment:
according to the task, you should avoid using loop and forEach. As an alternative, you can reduce method here

Reply:
also, arrow functions should not return assignments.

in general,  if you need to return the result of the subtraction you should just return it. 

but in this case, you have a loop and on each iteration, you just add a result to the sum variable. so in this case we need to add  braces and we can just omit return (any way it is ignored and not used anywhere):
```suggestion
  men.forEach((m) => {
    sum += m.died - m.born;
  });
```

also you can read more about this rule here: https://eslint.org/docs/latest/rules/no-return-assign